### PR TITLE
Making the pre-calculated fluid work again

### PR DESCRIPTION
**Pull request by GuillermoCasas**
Now it works fine and is not too expensive, as proven by the T-junction examples. Several bugs concerning the variable management options and some json modifications have been resolved.

### CODE REVIEW

File: applications/swimming_DEM_application/python_scripts/swimming_DEM_algorithm.py [hunk 1]
```diff
@@ -442,6 +444,8 @@ def FluidInitialize(self):
 
     def ReadFluidModelParts(self):
         self.fluid_algorithm.ReadFluidModelPart()
+        for node in self.fluid_model_part.Nodes:
+            print(self.fluid_model_part)
```
Comment (maceligueta) [inline applications/swimming_DEM_application/python_scripts/swimming_DEM_algorithm.py]: Are you sure you want to print this??

Comment (GuillermoCasas) [reply applications/swimming_DEM_application/python_scripts/swimming_DEM_algorithm.py]: Oops!

File: applications/swimming_DEM_application/python_scripts/variables_management.py [hunk 2]
```diff
@@ -9,6 +9,8 @@ def AddNodalVariables(model_part, variable_list):
 
     for var in variable_list:
         model_part.AddNodalSolutionStepVariable(var)
+        if model_part.Name == "FluidPart":
+            print(var)
```
Comment (maceligueta) [inline applications/swimming_DEM_application/python_scripts/variables_management.py]: This one?

Comment (GuillermoCasas) [reply applications/swimming_DEM_application/python_scripts/variables_management.py]: I just removed it

**Review by maceligueta — APPROVED** [review]

**Pull request merged**